Files/Helpers/ThemeHelper.cs

```diff
@@ -99,31 +62,32 @@ private static async void UiSettings_ColorValuesChanged(UISettings sender, objec
                 // Dispatch on UI thread so that we have a current appbar to access and change
                 await currentApplicationWindow.Dispatcher.RunAsync(Windows.UI.Core.CoreDispatcherPriority.High, () =>
                 {
-                    UpdateTheme();
+                    ApplyTheme();
                 });
             }
         }
 
-        public static void UpdateTheme()
+        private static void ApplyTheme()
         {
-            switch (RootTheme)
+            var rootTheme = RootTheme;
+
+            if (Window.Current.Content is FrameworkElement rootElement)
+            {
+                rootElement.RequestedTheme = rootTheme;
+            }
+
+            switch (rootTheme)
             {
                 case ElementTheme.Default:
                     App.AppSettings.AcrylicTheme.SetDefaultTheme();
-                    titleBar.ButtonHoverBackgroundColor = (Color)Application.Current.Resources["SystemBaseLowColor"];
```

Comment:
Check titlebar buttons color.
![image](https://user-images.githubusercontent.com/20501502/106055661-59d0a780-60f6-11eb-938e-f5f0ab91161b.png)

Reply:
@AndyMDoyle It looks like the code to set the title bar buttons color was removed in one of your commits.